helpers/queries/virtual-protocol-volume.sql

```
@@ -0,0 +1,127 @@
+        SELECT DISTINCT
+            varbinary_substring(topic1, 13, 20) as token,
+            varbinary_substring(topic2, 13, 20) as pair
+        FROM base.logs
+        WHERE topic0 IN (
+            0xb9ee8aa6d909a3efd0bf1b0bc2bde7f998f7ad30178b0d45f9227f5382cebc8f,
+            0x6ed5dc54f1333f448f2cdf7a6efc675343f880035d6f647fb7f6e9cbf8959718
+        )
+        AND block_time >= timestamp '2024-10-01'
+        AND block_time <= from_unixtime({{endTimestamp}})
+
+        UNION
```

SQLFluff (4.2.2)

[error] 'UNION [DISTINCT|ALL]' is preferred over just 'UNION'.

(AM02)